### ISSUE TITLE
[a11y] New Collection Object adds preparations automatically & cannot be removed

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1359]
**Describe the bug**
When creating a new collection object record, a preparation is already added and cannot be removed using the subview header. 

**To Reproduce**
1. Create a new CO record in the Tissue Collection
2. See preparation already visible
3. See no remove option in subview header

**Expected behavior**
No preparations should be present by default. There should be a **-** remove option as well.

**Screenshots**

Immediately after opening the entry form:
![image](https://user-images.githubusercontent.com/37256050/165967546-ab781e73-8567-4249-acc8-736a6b709974.png)

![image](https://user-images.githubusercontent.com/37256050/165967490-ba5658c0-75fe-4807-9172-78f059ae57f5.png)

macOS, Edge

**Comment by maxxxxxdlp** [comment]
Preparation is created automatically if you have a corresponding remote pref set:
<img width="1336" alt="Screen Shot 2022-04-29 at 12 58 38" src="https://user-images.githubusercontent.com/40512816/165998040-c3095eaa-54e1-48e1-a6b6-769916c1c6db.png">

There is no single delete button at the top because there can be multiple records in a form table. There is a trash bin button at the end of each row
